hieradata/dev/wso2/wso2is/5.1.0/default/default.yaml

```diff
@@ -77,63 +80,66 @@ wso2::clustering:
+#  wso2_config_db:
+#    name: WSO2_CONFIG_DB
+#    description: The datasource used for config registry
+#    driver_class_name: "%{hiera('wso2::datasources::mysql::driver_class_name')}"
+#    url: jdbc:mysql://192.168.100.1:3306/IS_DB?autoReconnect=true
+#    username: "%{hiera('wso2::datasources::common::username')}"
+#    password: "%{hiera('wso2::datasources::common::password')}"
+#    jndi_config: jdbc/WSO2_CONFIG_DB
+#    max_active: "%{hiera('wso2::datasources::common::max_active')}"
+#    max_wait: "%{hiera('wso2::datasources::common::max_wait')}"
+#    test_on_borrow: "%{hiera('wso2::datasources::common::test_on_borrow')}"
+#    default_auto_commit: "%{hiera('wso2::datasources::common::default_auto_commit')}"
+#    validation_query: "%{hiera('wso2::datasources::mysql::validation_query')}"
+#    validation_interval: "%{hiera('wso2::datasources::common::validation_interval')}"
 #
-#   wso2_gov_db:
-#     name: WSO2_GOV_DB
-#     description: The datasource used for gov registry
-#     driver_class_name: "%{hiera('wso2::datasources::mysql::driver_class_name')}"
-#     url: jdbc:mysql://mysql-govdb:3306/WSO2_GOV_DB?autoReconnect=true
-#     username: "%{hiera('wso2::datasources::common::username')}"
-#     password: "%{hiera('wso2::datasources::common::password')}"
-#     jndi_config: jdbc/WSO2_GOV_DB
-#     max_active: "%{hiera('wso2::datasources::common::max_active')}"
-#     max_wait: "%{hiera('wso2::datasources::common::max_wait')}"
-#     test_on_borrow: "%{hiera('wso2::datasources::common::test_on_borrow')}"
-#     default_auto_commit: "%{hiera('wso2::datasources::common::default_auto_commit')}"
-#     validation_query: "%{hiera('wso2::datasources::mysql::validation_query')}"
-#     validation_interval: "%{hiera('wso2::datasources::common::validation_interval')}"
-#
-#   wso2_user_db:
-#     name: WSO2_USER_DB
-#     description: The datasource is used for user management and userstore
-#     driver_class_name: "%{hiera('wso2::datasources::mysql::driver_class_name')}"
-#     url: jdbc:mysql://mysql-userdb:3306/WSO2_USER_DB?autoReconnect=true
-#     username: "%{hiera('wso2::datasources::common::username')}"
-#     password: "%{hiera('wso2::datasources::common::password')}"
-#     jndi_config: jdbc/WSO2USER_DB
-#     max_active: "%{hiera('wso2::datasources::common::max_active')}"
-#     max_wait: "%{hiera('wso2::datasources::common::max_wait')}"
-#     test_on_borrow: "%{hiera('wso2::datasources::common::test_on_borrow')}"
-#     default_auto_commit: "%{hiera('wso2::datasources::common::default_auto_commit')}"
-#     validation_query: "%{hiera('wso2::datasources::mysql::validation_query')}"
-#     validation_interval: "%{hiera('wso2::datasources::common::validation_interval')}"
+#  wso2_gov_db:
+#    name: WSO2_GOV_DB
+#    description: The datasource used for gov registry
+#    driver_class_name: "%{hiera('wso2::datasources::mysql::driver_class_name')}"
+#    url: jdbc:mysql://192.168.100.1:3306/WSO2REG_DB?autoReconnect=true
```

Comment:
When we are testing default platform using puppet modules vagrant setup, this is the IP address set to local machine and we are creating dbs in our local machine and trying to connect to them via  this IP. This is the IP address used in all products db url in default platform hiera data yaml files.